### DISCUSSION TITLE
机器人部署出问题了 BUG？

### DISCUSSION THREAD

**Discussion by 1485470102** [Arondight/Adachi-BOT#711] [Q&A] [answered]
一开始还有用 我重启一下机器人 然后发命令就没反应了 怎么解决呢
![image](https://user-images.githubusercontent.com/58935312/158065317-e183595d-1751-4ee3-9d8c-3ba5b014f2cf.png)
![image](https://user-images.githubusercontent.com/58935312/158065335-b77dd320-3b81-41c5-a437-6889c9e10083.png)

**Comment by mark9804** [comment, marked as answer]
cookie 格式错了
```yaml
# ==============================================================================
# 此为项目的米游社 Cookie 配置。
# ==============================================================================
# 将此文件复制到 ../config/ 下进行修改，不要直接修改此文件。
# ==============================================================================
#
# 1. 按照注释的示例配置若干个 Cookie 并删掉行首可能存在的注释符号（ # ）。
# 2. 当前（2021-10-09），每个 Cookie 每天最多查询 30 个不同用户的信息。
# 3. 注释中的示例格式根据实际情况可能会有略微变更。
#
cookies:
  #- UM_distinctid=...; _ga=...; _gid=...; CNZZDATA1275023096=...; _MHYUUID=...; ltoken=...; ltuid=...; cookie_token=...; account_id=...
  - 把这一行换成你的cookie

```

**Reply by 1485470102** [reply]
<details>

![image](https://user-images.githubusercontent.com/58935312/158065237-f0fa4ded-4e09-4cbd-980c-22c6b310408d.png)
![image](https://user-images.githubusercontent.com/58935312/158065249-3ba17a1e-3256-4cbe-b8a3-c38bdf5b2fec.png)
</details>

我知道了 我配置米游社ck后就没反应 不配置就有反应 但是不配置米游社ck查不了信息

**Reply by mark9804** [reply]
cookie 格式错了
```yaml
# ==============================================================================
# 此为项目的米游社 Cookie 配置。
# ==============================================================================
# 将此文件复制到 ../config/ 下进行修改，不要直接修改此文件。
# ==============================================================================
#
# 1. 按照注释的示例配置若干个 Cookie 并删掉行首可能存在的注释符号（ # ）。
# 2. 当前（2021-10-09），每个 Cookie 每天最多查询 30 个不同用户的信息。
# 3. 注释中的示例格式根据实际情况可能会有略微变更。
#
cookies:
  #- UM_distinctid=...; _ga=...; _gid=...; CNZZDATA1275023096=...; _MHYUUID=...; ltoken=...; ltuid=...; cookie_token=...; account_id=...
  - 把这一行换成你的cookie

```

**Reply by 1485470102** [reply]
> cookie 格式错了
> 
> ```yaml
> # ==============================================================================
> # 此为项目的米游社 Cookie 配置。
> # ==============================================================================
> # 将此文件复制到 ../config/ 下进行修改，不要直接修改此文件。
> # ==============================================================================
> #
> # 1. 按照注释的示例配置若干个 Cookie 并删掉行首可能存在的注释符号（ # ）。
> # 2. 当前（2021-10-09），每个 Cookie 每天最多查询 30 个不同用户的信息。
> # 3. 注释中的示例格式根据实际情况可能会有略微变更。
> #
> cookies:
>   #- UM_distinctid=...; _ga=...; _gid=...; CNZZDATA1275023096=...; _MHYUUID=...; ltoken=...; ltuid=...; cookie_token=...; account_id=...
>   - 把这一行换成你的cookie
> ```

谢谢 我解决了
